php/WC_Gateway_Blockonomics.php

```diff
@@ -108,42 +104,8 @@ public function process_payment($order_id)
         $order_key = method_exists( $order, 'get_order_key' ) ? $order->get_order_key() : $order->order_key;
         $cancel_url = add_query_arg('order_key', $order_key, $cancel_url);
 
-        $blockonomics = new Blockonomics;
```

Comment:
Also just a question to understand better, what was the reason of moving this code?